cronjobs.yml

```diff
@@ -9,12 +9,40 @@ cronJobs:
   - name: process-test-mappings
     schedule: "0 1 * * *"
     command: ["/bin/sh", "cronjobs/process_test_mapping_work_items.sh"]
+    resources: # Burstable
+      requests: # guaranteed amount of resources
+        cpu: 500m
+        memory: 6Gi
```

Comment:
These requests and limits are huge. Why do we need so much memory?

Reply:
I misread the maxes in TIG-2051 as the recommendations, I'll recalibrate.

Reply:
This comment has now been addressed. I'm going to merge (and thus deploy the changes). We can revisit the limits if there is an issue.